.github/workflows/new-release.yml

```diff
@@ -8,38 +8,24 @@ jobs:
   make-new-release:
     name: "Releasing plugin"
     runs-on: ubuntu-latest
+    if: github.repository == 'JLLeitschuh/ktlint-gradle'
+    env:
+      GRADLE_PUBLISH_KEY: ${{ secrets.GRADLE_PUBLISH_KEY }}
+      GRADLE_PUBLISH_SECRET: ${{ secrets.GRADLE_PUBLISH_SECRET }}
+      GITHUB_KEY: ${{ secrets.GithubKey }}
```

Comment:
Why move these to the top when they are only needed for 2 steps. I don't see value in exposing these env variables to other action steps.

Reply:
Just reuse them for steps.